README.md

```diff
@@ -62,6 +62,52 @@ Contacts.getContact( contactIdentifier, (error, contact) =>  {
 });
 ```
 
+#### Delete a Single Contact
+
+```js
+let contactIdentifier = 'A7806266-6574-4731-82E1-C54946F63E1C';
+
+Contacts.deleteContact( contactIndetifier, (error, contactId) => {
```

Comment:
Why is the `contactId` being returned. Shouldn't that just be a Boolean response. `true` for success and `false` for some kind of error?

Unless I'm missing some value in returning the `contactId`.